src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (this.setKey));
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', (this.setKey));
+  }
+
+  setKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {pressedKey === ''
+            ? 'Nothing was pressed yet'
```

Comment:
You could just check if `pressedKey` (don't need this part `=== ' ' ` )

Reply:
Agreed, fixed it right away, thank you for the review 😄